packages/medusa/src/strategies/batch-jobs/product/export.ts

```diff
@@ -449,4 +458,68 @@ export default class ProductExportStrategy extends AbstractBatchJobStrategy<
         },
       })
   }
+
+  /**
+   * Return the maximun number of each relation that must appears in the export.
```

Comment:
question: @srindom wdyt of the description?

Reply:
👍